cores/esp8266/core_esp8266_i2s.c

```diff
@@ -148,8 +157,14 @@ void ICACHE_FLASH_ATTR i2s_slc_end(){
   SLCRXL &= ~(SLCRXLAM << SLCRXLA); // clear RX descriptor address
 
   for (int x = 0; x<SLC_BUF_CNT; x++) {
-	  free(i2s_slc_buf_pntr[x]);
+    free(i2s_slc_buf_pntr[x]);
```

Comment:
The ptrs here should be nulled after free.

Reply:
Actually, scratch that.  i2s_slc_buf_pntr[] is a dynamically allocated array of pointers.  I free each pointer i2s_slc_buf_pntr[0...n], then completely free and null i2s_slc_buf_pntr.  So there is no way to get to those old pointers, ever, so no need to NULL.

Reply:
What I am guarding against is a second call to begin() without a call to end(), I.e. make sure no leaks. This seems to be common usage in ArduinoLand.

Reply:
Arduinoland is a dangerous place to live...

It'll still (randomly, making it even more fun) crash w/o the changes I mentioned on the i2s_begin() call due to ISR and DMA working w/the old addresses.